css/index.styl

```diff
@@ -1406,9 +1406,5 @@ _:-ms-fullscreen, :root .auth0-lock.auth0-lock .auth0-lock-content-wrapper
   flex-basis: auto !important
   max-height: 70vh
 
-// iPhone display fix - https://github.com/auth0/lock/issues/1588
-.auth0-lock-iphone.auth0-lock .auth0-lock-form
-  display: initial;
-
-.auth0-lock-iphone.auth0-lock .auth0-lock-content
-  padding: 20px;
+.auth0-lock.auth0-lock .auth0-lock-form
+  animation: none
```

Comment:
Could you remove these instead? https://github.com/auth0/lock/blob/0e236b53d00c6744b640513a8d15fe20fb71a5fa/css/index.styl#L324-L325

Reply:
Ah yes - much better